frontend/src/components/tables/MarketTables.jsx

```diff
@@ -41,40 +34,48 @@ const formatResolutionDate = (resolutionDateTime) => {
 };
 
 const MarketRow = ({ marketData }) => (
-  <tr>
-    <td className='px-6 py-4 text-white'>
-      <Link to={`/markets/${marketData.market.id}`}>⬆️ ⬇️</Link>
+  <tr className='hover:bg-gray-700 transition-colors duration-200'>
+    <td className='px-6 py-4 whitespace-nowrap'>
+      <Link
+        to={`/markets/${marketData.market.id}`}
+        className='text-blue-400 hover:text-blue-300'
+      >
+        ⬆️⬇️
+      </Link>
     </td>
-    <td className='px-6 py-4 text-sm text-gray-500'>
+    <td className='px-6 py-4 whitespace-nowrap text-sm text-gray-300'>
       {marketData.lastProbability.toFixed(3)}
     </td>
-    <td className='px-6 py-4 text-sm font-mono text-gray-500'>
+    <td className='px-6 py-4 whitespace-nowrap text-sm font-medium text-gray-300'>
       <Link
         to={`/markets/${marketData.market.id}`}
-        className='text-blue-600 hover:text-blue-800'
+        className='hover:text-blue-400 transition-colors duration-200 block max-w-xs overflow-hidden overflow-ellipsis'
+        title={marketData.market.questionTitle}
```

Comment:
OK, while on desktop the overlap problem seems to have been solved, on mobile we have to scale over to the right.

For now, should we maybe just show minimal information, like the Trade Buttons, Title and Price?

![image](https://github.com/user-attachments/assets/87526d62-a9d1-48a4-b32b-00528346c00e)

In the future maybe we can re-design this to display more information either as a call out or pop or something...?

Reply:
Yeah idk what to do with mobile for it to still make sense, title in this layout will pretty much always be cut off so we need a completely different mobile layout.

I'll look into it.

Reply:
Fixed here https://github.com/openpredictionmarkets/socialpredict/pull/242/commits/016cf16649d5b48356ec852ba472e269e845fdfd

Reply:
OK what you did looks great.

I'm noticing with the issue above, the reason for the problem with the overlap is that it's not just 160 characters, it's a 160 character word, which should be unexpected. I guess I don't know if there is a way to text wrap long words. We'll call this an optimization and just approve everything for now.